cachito/web/utils.py

```diff
@@ -63,3 +65,23 @@ def str_to_bool(item):
         return item.lower() in ("true", "1")
     else:
         return False
+
+
+def deep_sort(orig_item):
+    """
+    Return a new element recursively sorted in ascending order.
+
+    :param orig_item: Original dict or list
+    :return: Recursively sorted dict or list according to orig_item
```

Comment:
Can we add `:rtype:` separately just to be consistent with the rest of the file?

Reply:
@nirzari I am afraid that return type can be `list` and `OrderedDict` at the same time.

Reply:
IMO the return type is basically `Any`, if you pass in something that is neither a dict nor a list, you get it back unchanged

Reply:
Done. Thank you! @chmeliik @nirzari PTAL :)

Reply:
Yeah I just meant adding another line for return type. Looks good now!